README.md

```diff
@@ -74,7 +74,7 @@ Once you have confirmed access, you can run the following command to download th
 ```bash
 # Get your HF token from https://huggingface.co/settings/tokens
 
-# llama3 tokenizer.model
+# llama3 or 3.1 tokenizer.model
```

Comment:
Upon comparision, the tokenizer is same across model size and llama 3 and llama 3.1